app/src/app/search/search.component.ts

```diff
@@ -169,7 +146,7 @@ export class SearchComponent implements OnInit, OnDestroy {
             relativeTo: this.route,
             queryParams: { limit: this.limit },
             queryParamsHandling: 'merge'
-          });
+          }).then();
```

Comment:
Is the promise resolving necessary here?

Reply:
No not really. Does my IDE automatically